spec/features/admin_visit_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+feature 'admin visits paths' do
+  let(:user) { create(:admin) }
+  let(:album) { create(:album) }
+  let(:cafe_work) { create(:cafe_work) }
+
+  controllers = [:albums, :cafe_works, :constants, :contacts, :councils,
+                 :documents, :elections, :events, :faqs, :menus, :news,
+                 :notices, :pages, :posts, :rents, :work_posts]
+  paths = {
+    albums: [ :show ],
+    cafe_works: [ :index, :nyckelpiga ],
+    constants: [ :index, :new, :show ],
+    contacts: [ :index, :new, :show],
+    councils: [ :index, :show ],
+   #documents: [ :index, :new ],
```

Comment:
Incorrect indentation detected (column 3 instead of 4).<br>Missing space after #.